doc/src/extras.rst

```diff
@@ -141,6 +141,383 @@ Logging cursor
 
```

Comment:
Quick start for docs:
---

You must be using PostgreSQL 9.4 or above to run this quick start.

Make sure that `replication` connections are permitted for user `postgres` from `localhost` in `pg_hba.conf` then `SELECT pg_reload_conf()`. Create a database `psycopg2test`.

Then run something like the following to quickly try the replication support out. This is not production code - it has no error handling, it sends feedback too often, etc - and it's only intended as a simple demo of logical replication functionality.

```
import psycopg2
import psycopg2.extras

conn = psycopg2.connect('dbname=psycopg2test user=postgres',
   connection_factory=psycopg2.extras.LogicalReplicationConnection)
cur = conn.cursor()
try:
    cur.start_replication(slot_name = 'pytest')
except psycopg2.ProgrammingError:
    cur.create_replication_slot('pytest',
        slot_type=psycopg2.extras.REPLICATION_LOGICAL,
        output_plugin='test_decoding')
    cur.start_replication(slot_name = 'pytest')

class DemoConsumer(object):
    def consume(self, msg):
        print msg.payload
        msg.cursor.send_replication_feedback(flush_lsn=msg.data_start)

democonsumer = DemoConsumer()

cur.consume_replication_stream(democonsumer)
```

This will continue running until cancelled with control-C.

You can now make changes to the `psycopg2test` database using a normal `psycopg2` session, `psql`, etc and see the logical decoding stream printed by this demo client.